docs/development.rst

```diff
@@ -17,7 +17,7 @@ Contributions
 
 Some guidance for contributors:
 
-- discuss about changes on github issue tracker, IRC or mailing list
+- discuss changes on the GitHub issue tracker, IRC or on the mailing list
```

Comment:
I would rather also say "on IRC" here... But I don't really feel strongly about it :)